src/app/components/MediaLoader/index.styles.ts

```diff
@@ -1,9 +1,12 @@
 import { css, Theme } from '@emotion/react';
 
 export default {
-  figure: css({
-    margin: 0,
-  }),
+  figure: ({ spacings }: Theme) =>
+    css({
+      margin: 0,
+      paddingBottom: `${spacings.TRIPLE}rem`,
+      width: '100%',
+    }),
```

Comment:
Copied from src/app/components/ImageWithCaption/index.styles.ts

We actually override the default padding (from 24px to 16px) on the Live Page so this isn't actually required for this PR. But thought maybe I shoud fix it since this would cause the same issue on article pages when we implement the Media Loader here.

Reply:
Just thinking should we use `marginBottom` here instead since we want to extend the gap between the separate elements, rather than making the `figure` itself appear to have more space below it?

Reply:
Ah ok, maybe best to leave it then. I wonder why padding was used originally for paragraphs. Margin is used to add space outside the element, whereas padding is for inside.

Reply:
Hmm yh I agree - it does feel worth looking into